### ISSUE TITLE
Revert CORS change commit due to service's bias towards some users/countries.

### ISSUE THREAD

**Issue by Raccffy** [CodeZombie/TypeRip#50]
Revert [CORS change commit](https://github.com/CodeZombie/TypeRip/commit/2b0e4bb1fc8b75dd78b5e50aae8fb072d7fbcf0d), because [corsproxy.io](corsproxy.io) (underlying service for resolving CORS error) is biased towards Russian, proxy and Tor users.

Old CORS service works well and *should be used instead.*

### Steps to reproduce

1. Open free web proxy (like [this one](https://coproxy.io/free-web-proxy)).
2. Load TypeRip page, copy any valid Adobe Font URL and try to fetch font data.

### Expected behavior

Font was fetched successfully.

### Observed behavior

TypeRip returned "Request Failed", because underlying CORS service throwed 403 (forbidden) error.

#### Bug reproduction

*Note: Besides CORS service "betrayal", I also demonstrated, that the old one is still up and running. This is done by overriding the script with replaced malicious URL.*

https://github.com/user-attachments/assets/ba8c3afa-5a8d-4d56-93a6-dc26498cd277

#### Service's message

Attempting to use CORS service using Russian IP address or Tor returns this lovely message:

![Country blocked!](https://github.com/user-attachments/assets/e2fa06e3-d84e-4c2a-82a3-5b941b9ec87e)

Attempting to use example free web proxy invokes Cloudflare's "protection measures".

**Comment by CodeZombie** [comment]
Good catch, thanks for the detailed issue. My apologies for switching over to a proxy without thoroughly researching it.

I've added a new feature that will "cycle" through a list of cors proxies until it finds one that works and store that one in your browser to use for all future network requests.
I'm keeping `corsproxy.io` in the list because it's very fast compared to `api.allorigins.win` from my testing, but I've added `api.allorigins.win` into the list as well, so your browser should use that from now on when you use the tool.

Let me know if it doesn't work!!!
